src/main/scala/viper/gobra/frontend/info/implementation/typing/ghost/GhostMemberTyping.scala

```diff
@@ -99,10 +99,30 @@ trait GhostMemberTyping extends BaseTyping { this: TypeInfoImpl =>
     } else noMessages
   }
 
+  private[typing] def wellDefIfOpaqueFunction(member: PFunctionDecl): Messages = {
```

Comment:
wellDefIfOpaqueFunction/Method and isPureFunction/isPureMethod are kind of useless, as we ensure in wellDefSpec that opaque must come with pure. There is also no test case that can exercise these errors, as they get caught by wellDefSpec. Should I delete this?